src/types.mligo

```diff
@@ -223,6 +234,7 @@ type forbid_xtz_params =
   | Freeze of freeze_param
   | Unfreeze of unfreeze_param
   | Update_delegate of update_delegate_params
+  | Unfreeze_vote of unfreeze_vote_param
```

Comment:
Why is this called `Unfreeze` when what it is doing seems to be just unstaking staked frozen tokens back to unstaked frozen tokens. Is any 'Unfreez' ing happening here?

Reply:
That's true, the term `unfreeze` on `flush` was used a lot previously, but the correct term should be `unstake` on `flush`, since we have a separate `unfreeze` entrypoint now. 

I'll rename this to `unstake` instead.

Reply:
I've pushed the changes, have to squash the commits due to fixing conflicts when rebasing.